efcsydney/efcsydney-roster
# #453 Reduce the JavaScript bundle size by code splitting

#### Description

Use `loadable-component` to load chunk JavaScript files according to the path.

#### QA URL

* https://demo-roster.efcsydney.org/#/index/english
* https://demo-roster.efcsydney.org/#/admin/services

#### QA Steps

1. Open the Chrome Developer Tool
1. Load the different URLs above
1. Check the amount and size of JS files. 

Ensure it loads the JS files according to different pages.
